src/app/components/auth/auth.component.ts

```diff
@@ -18,7 +18,7 @@ export class AuthComponent implements OnInit {
    */
   localRouter: any;
 
-  @ViewChild('authContainer') authContainer: ElementRef;
+  @ViewChild('authContainer', {read: null}) authContainer: ElementRef;
```

Comment:
Hey @drepram Why we need this change here? If not necessary then can you please remove?

Reply:
![Screen Shot 2020-01-06 at 22 35 51](https://user-images.githubusercontent.com/34530026/71828478-00ee3580-30d5-11ea-9549-e4bc91259c6a.png)
![Screen Shot 2020-01-06 at 22 36 27](https://user-images.githubusercontent.com/34530026/71828480-00ee3580-30d5-11ea-8753-eafca9d5218d.png)

It's necessary, if the opts parameter is not provided, the build will fail.

Reply:
It's a bit tricky with the static field, one version wants it, the other one rejects it. I follow the latter, the one that Travis is using.

Reply:
Changed!

Reply:
I see... if the build is failing without this on the master branch then please open a separate PR for this. Please remove this and the one below as it is not related to this PR.